server/src/test/java/umm3601/mongotest/MongoSpec.java

```diff
@@ -34,15 +46,21 @@
  * To test "our" code we'd want the tests to confirm that
  * the behavior of methods in things like the UserController
  * do the "right" thing.
- * <p>
- * Created by mcphee on 20/2/17.
  */
+// The tests here include a ton of "magic numbers" (numeric constants).
+// It wasn't clear to me that giving all of them names would actually
+// help things. The fact that it wasn't obvious what to call some
+// of them says a lot. Maybe what this ultimately means is that
+// these tests can/should be restructured so the constants (there are
+// also a lot of "magic strings" that Checkstyle doesn't actually
+// flag as a problem) make more sense.
```

Comment:
I certainly have seen test classes that start by defining thirty different constants for sample data—but I agree with you that we're probably okay if we don't do that! 🙂